vocab.json

```diff
@@ -671,6 +671,60 @@
                 {"id": 15,
                 "spanish": ["viejo", "antiguo", "antigua"],
                 "english": ["old"]}
+            ]
+  },
+  {
+    "id": 9,
+    "name": "List 10 - House and Home 2",
+    "words": [
+                {"id": 1,
```

Comment:
I have used zero-based indexing for other lists - no need to change it as it won't make any difference, but it's worth being aware of as it comes up quite often when iterating through arrays, which are generally zero-based in the languages we use - it makes it easier to use `index` rather than `index + 1`.